<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonEditor</class>
 <widget class="QDialog" name="PersonEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Operation</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="operation">
     <item>
      <property name="text">
       <string>Ignore</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Show</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hide</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="person_box">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="fade_time">
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QComboBox" name="person">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="appear_combobox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>from_left</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>from_right</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>from_top</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>from_bottom</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QCheckBox" name="appear_backwards">
        <property name="text">
         <string>Backwards</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Person</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QComboBox" name="mood">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QDoubleSpinBox" name="appear_time">
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="fade_checkbox">
        <property name="text">
         <string>Fade</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Mood</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="appear_checkbox">
        <property name="text">
         <string>Appear</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QCheckBox" name="fade_backwards">
        <property name="text">
         <string>Backwards</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QComboBox" name="comboBox_4">
     <item>
      <property name="text">
       <string>left</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>middle-left</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>middle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>middle-right</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>right</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
